app/views/dev_mode.py

```diff
@@ -6,13 +6,17 @@
 from flask import Blueprint, redirect, render_template, request, current_app
 from structlog import get_logger
 
-from app.cryptography.jwt_encoder import Encoder
+from app.cryptography.token_helper import encrypt_jwe, encode_jwt
+from app.secrets import PURPOSE_AUTHENTICATION
 from app.utilities.schema import available_json_schemas
 
 # pylint: disable=too-many-locals
 logger = get_logger()
 dev_mode_blueprint = Blueprint('dev_mode', __name__, template_folder='templates')
 
+EQ_USER_AUTHENTICATION_RRM_PRIVATE_KEY_KID = "709eb42cfee5570058ce0711f730bfbb7d4c8ade"
```

Comment:
Do we have examples in the ons-schema-docs of the extract format of the kid like this, i.e. a lowercase alpha-numeric byte (0-9a-f) string exactly (?) characters long?